packages/skeleton-react/vite.config.ts

```diff
@@ -1,19 +1,37 @@
-import { defineConfig } from "vite";
-import path from "node:path";
+import { defineConfig } from "vitest/config";
```

Comment:
you actually don't need to change the import from 'vite' to 'vitest' since you are in the vite.config.ts file.
I'd prefer to use the 'vite' import if they update something important and 'vitest' lacks behind with updating their config

Reply:
Tell that to Typescript! Red squiggles until I modified this in the project.

Reply:
Interesting... I'll look at it tomorrow alongside the userEvent stuff

Reply:
@TazorDE just a reminder about this issue! When I revert the change to default settings the `defineConig.test` is not supported. The tests functionally run, but there appears to be a Type error.